dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -88,4 +88,7 @@ certCheckerErrorSchedule=Certificate checker error schedule
 certCheckerErrorSchedule.desc=Performs the same function as the certCheckerSchedule attribute, but on an alternate schedule. For example, the certCheckerErrorSchedule attribute can be set to a shorter interval than the certCheckerSchedule attribute, to increase the frequency of checks after a failed request. The interval from the certCheckerSchedule attribute is resumed after the certificate is renewed. 
 
 disableMinRenewWindow=Disable minimum renew window
-disableMinRenewWindow.desc=Enables immediate certificate renew requests by disabling the specified minimum amount of time between renew requests. The minimum renew time is to 15 seconds.
\ No newline at end of file
+disableMinRenewWindow.desc=Enables immediate certificate renew requests by disabling the specified minimum amount of time between renew requests. The minimum renew time is to 15 seconds.
```

Comment:
I don't think you changed this one, but I'm not sure I understand "The minimum renew time is to 15 seconds." Does this mean "The minimum renew time is 15 seconds."?

Reply:
True, that appears to have an extra word!